src/mapi/glapi/gen/es_EXT.xml

```diff
@@ -795,6 +795,14 @@
     <enum name="FRAGMENT_SHADER_DISCARDS_SAMPLES_EXT" value="0x8A52"/>
 </category>
 
+<!-- 150. GL_EXT_sRGB -->
```

Comment:
Initial impression is that this looks good!

I will have a closer look tomorrow at the conditions added, I believe the genmipmap one could just be a check against gles2 context + extension (instead of !gles3 !desktop and extension) .. I believe shortest version would be just to check against ctx->API == API_OPENGLES2.